lib/ruby_lsp/requests/code_action_resolve.rb

```diff
@@ -51,7 +51,9 @@ def run
         scanner = @document.create_scanner
         start_index = scanner.find_char_position(source_range[:start])
         end_index = scanner.find_char_position(source_range[:end])
-        extracted_source = T.must(@document.source[start_index...end_index])
+        extracted_source = @document.source[start_index...end_index]
+
+        return Error::InvalidTargetRange if extracted_source.nil?
```

Comment:
Why is this needed? Since `code_action_resolve` requires the user to select something in the editor, I don't think the range would ever be `nil`.

Maybe this needs an improvement in the test instead?